### ISSUE TITLE
Code for generating training data from COCO Captions

### ISSUE THREAD

**Issue by shehanmunasinghe** [facebookresearch/ov-seg#7]
Could you please release the code for generating training data from COCO Captions and fine-tuning CLIP with the collected mask-category pairs?

**Comment by Jeff-LiangF** [comment]
Hi @shehanmunasinghe ,

We don't plan to release this part. However, we believe it is not hard to implement. Some hints: You may want to use the [regions](https://github.com/facebookresearch/ov-seg/blob/main/open_vocab_seg/ovseg_model.py#LL212C20-L212C27) which stands for the generated proposals. For noun extraction, we use [nltk](https://www.nltk.org/). Some details that need to pay attention to: make sure the input to CLIP is normalized and the saved images should be unit8. Saved data should follow [this format](https://github.com/facebookresearch/ov-seg/tree/main/open_clip_training#data) so it can be accepted in [open clip training](https://github.com/facebookresearch/ov-seg/tree/main/open_clip_training)
